swagger-parser/src/test/resources/examples/required_parametrs.yaml

```diff
@@ -0,0 +1,26 @@
+---
+swagger: '2.0'
+info:
+  version: 0.0.0
+  title: Simple API
+paths:
+  /:
+    get:
+      parameters:
+        - name: test1
+          in: query
+          required: true
+          type: string
+        - name: test2
+          in: body
+          schema:
+            type: object
+            required:
+              - test3
```

Comment:
maybe we could have a parameter with two validations? Required + regexp, just to check that we still generate sequences of validators as it is required